.github/validation_samples/ecal_pn/config.py

```diff
@@ -199,6 +194,22 @@
 ecalVeto = ecal_vetos.EcalVetoProcessor()
 ecalVeto.recoil_from_tracking = True
 
+# The Tracking modules produce a lot of helpful messages
+# but (at the debug level) is too much for commiting the gold log
+# into the git working tree on GitHub
+p.termLogLevel = 0
```

Comment:
Do you want to use the newer syntax? (I also want to double check that #1458 works.)
```suggestion
p.logger.termLevel = 0
```